roles/cobbler/templates/triggers/install/post/cephlab_ansible.sh

```diff
@@ -13,7 +14,15 @@ nmap -sT -oG - -p 22 $name | grep 22/open
 
 mkdir -p /var/log/ansible
 
+if [ $SECRETS_REPO_NAME != 'UNDEFINED' ]
```

Comment:
I think it might be cleaner to use jinja2 templates to handle this condition.  If ``secrets_repo.name == false`` then you could just not include that bit of code at all.  I've done something similar here: https://github.com/ceph/ceph-cm-ansible/blob/master/roles/cobbler/templates/snippets/cephlab_ubuntu_network#L29

I don't want to be pedantic, but I do think this is a good use of jinja2.  Thoughts?